ethsigner/signer/multiplatform/src/test/resources/metadata-toml-configs/azureconfig.toml

```diff
@@ -0,0 +1,11 @@
+[metadata]
```

Comment:
Is this file used for testing? I could see where

Reply:
Missed a word here. Meant to say, I can't see where this file is being used.

Reply:
it is now used for testing